src/components/IconButton/constants.ts

```diff
@@ -0,0 +1,28 @@
+import { Color, Size, Variant } from '@/types';
+
+export const IconButtonColor = {
+  ERROR: Color.ERROR,
+  INFO: Color.INFO,
```

Comment:
is this one "primary"? or should we add it?

Reply:
I don't think we implement a `primary` button following the convention of the other colors. What we have been calling `primary` seemed to fit more into a `neutral` color since its just gray whereas `primary` in our Tailwind colors is a blue color. Thats a good callout though, I don't feel strongly on the topic but feel like it should get normalized (i.e. If our primary color is going to blacks and grays, our Tailwind color for primary should match that)

Reply:
And to answer your initial question, `Coor.INFO` is the blue colors